cPP_DSC.adoc

```diff
@@ -1332,6 +1374,10 @@ FCS_OTV_EXT.1.1:: The TSF shall perform _cryptographic one-time value generation
 |NIST SP 800-108, NIST SP 800-135r1
 |Salts and IVs as directed in use of HMAC and AES modes. Please reference the relevant standards.
 
+|PBKDF
```

Comment:
The application note below mentions PBKDF, but it wasn't actually listed here in the table.

Reply:
Resolved in #308.